docs/UserGuide.md

```diff
@@ -101,61 +98,33 @@ Format: `help`
 
 ### Adding an employee: `add`
 
-Adds employees into the EmployeeManager application. 
-Employee ID can either be specified or not. If specified, the application will check the existing list of employees 
-to check whether the entered ID is unique. If not unique it will show an error. If no Employer ID is specified then the 
-application will automatically assign a unique ID to the employee.
+Adds an employee to EmployeeManager.
 
-Format: `add [id/EMPLOYEE_ID] name/NAME position/POSITION department/DEPARTMENT salary/SALARY`
+Format: `add i/ID n/NAME p/PHONE e/EMAIL d/DEPARTMENT r/ROLE s/SALARY`
 
 Examples:
-* `add id/A035B name/Richard Hew Jia Liang position/Supervisor department/Marketing salary/5000`
-* `add name/Richard Hew Jia Liang position/Supervisor department/Marketing salary/5000`
-
-Constraints:
-* Employee ID must be unique if specified and up to 10 characters
-* Salary must be numerical and >= 0
-* Name, Position, and Department up to 100 characters each
+* `add i/A00001 n/John Doe p/97328601 e/johndoe@example.com d/Marketing r/Manager s/5000` Adds an employee `John Doe` into the list.
+* `add i/A00002 n/Betsy Crowe p/82517642 e/betsycrowe@example.com d/Finance r/Intern s/1200` Adds an employee `Betsy Crowe` into the list.
 
-Expected output when employee added successfully:
-* `Employee EMPLOYEE_ID NAME successfully added!`
+### Listing all employees : `list`
 
-Expected output when add command fails:
-* Invalid Salary : `Please input a non-negative salary as a numerical value (e.g. 4000)`
```

Comment:
Should we keep the expected output too?

Reply:
I feel like the expected output is not needed as it doesn't really help the user to use the application properly (plus past year user guides did not include the expected outputs). The constraints probably also give similar information as the expected output when the command fails.